core/scripts/bareos-config-lib.sh.in

```diff
@@ -997,15 +997,15 @@ apply_dbconfig_settings()
 get_local_hostname()
 {
     # put actual short hostname in configuration files
-    # try to get short hostname
-    hname=$(hostname -s)
+    if [ -n "$(command -v uname)" ]; then
+        hname=$(uname -n)
+    fi
+    if [ -z "${hname}" ] && [ -f '/etc/hostname' ]; then
+        hname="$(cat '/etc/hostname')"
+    fi
     if [ -z "${hname}" ]; then
-        # try to get long hostname
-        hname=$(hostname|sed 's/\..*//g')
-        if [ -z "${hname}" ]; then
-            # set to "localhost"
-            hname='localhost'
-        fi
+        # set to "localhost"
+        hname='localhost'
```

Comment:
this is a functional change: if you have the fqdn configured as hostname, you'll now get that. Previously, you always got the unqualified hostname.
If we agree that we want to have that change, I'm fine with it. We just shouldn't change it accidentially.

Reply:
Thanks, my bad I forget to strip done the unqualified hostname. I will take care of that one.